apps/codeforafrica/src/components/Footer/FooterDescription.js

```diff
@@ -0,0 +1,34 @@
+import { Typography } from "@mui/material";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import cfaIcon from "@/codeforafrica/assets/images/CfAlogoBW.png";
+
+function FooterDescription({ description }) {
+  if (!description) {
+    return null;
+  }
+
+  return (
+    <>
+      <Image src={cfaIcon} alt="" />
```

Comment:
We should at least use `ImageButton` or `Link` to make this link to the homepage.